services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/models/resource_tracker_container.py

```diff
@@ -0,0 +1,27 @@
+from arrow import Arrow
+from models_library.users import UserID
+from pydantic import BaseModel
+
+
+class ContainerResourceUsageMetric(BaseModel):
+    container_id: str
+    image: str
+    user_id: UserID
+    product_name: str
```

Comment:
I think we have a type for product name somewhere.

Reply:
thanks! changed to ProductName